_data/talks.yml

```
@@ -3,374 +3,437 @@
   track: Bengaluru
   date: 3rd Oct
-  time: 17:50 pm
+  time: 17:50 IST
   speaker: Abhijit Gadgil
   hyperlink: https://in.pycon.org/cfp/2020/proposals/python-and-cffi-visualizing-network-traces~aAy9a/
 
 - title: Resource Utilization as a Metric for Machine Learning
   track: Chennai
   date: 3rd Oct
-  time: 17:50 pm
+  time: 17:50 IST
   speaker: Akshay Bahadur
   hyperlink: https://in.pycon.org/cfp/2020/proposals/resource-utilization-as-a-metric-for-machine-learning~aKP8b/
 
 - title: Logging the right way!
   track: Hyderabad
   date: 3rd Oct
-  time: 17:50 pm
+  time: 17:50 IST
   speaker: Devi A S L
   hyperlink: https://in.pycon.org/cfp/2020/proposals/logging-the-right-way~dPN11/
 
 - title:
   track: Delhi
   date: 3rd Oct
-  time: 17:50 pm
+  time: 17:50 IST
   speaker:
 
 - title: Narrative-focused video games development with Ren'Py, an open source engine
   track: Bengaluru
   date: 3rd Oct
-  time: 18:25 pm
+  time: 18:25 IST
   speaker: Susan Shu Chang
   hyperlink: https://in.pycon.org/cfp/2020/proposals/narrative-focused-video-games-development-with-renpy-an-open-source-engine~az0qd/
 
 - title: Awesome Commandline Tools
   track: Chennai
   date: 3rd Oct
-  time: 18:25 pm
+  time: 18:25 IST
   speaker: Amjith Ramanujam
   hyperlink: https://in.pycon.org/cfp/2020/proposals/awesome-commandline-tools~eZlQb/
 
 - title: How Decorators Function
   track: Hyderabad
   date: 3rd Oct
-  time: 18:25 pm
+  time: 18:25 IST
   speaker: Andrew Knight
   hyperlink: https://in.pycon.org/cfp/2020/proposals/how-decorators-function~bYEp2/
 
 - title: Reproducible Scalable Workflows with Nix, Papermill and Renku
   track: Delhi
   date: 3rd Oct
-  time: 18:25 pm
+  time: 18:25 IST
   speaker: Rohit Goswami
   hyperlink: https://in.pycon.org/cfp/2020/proposals/reproducible-scalable-workflows-with-nix-papermill-and-renku~dNkxD/
 
 - title: Keynote
   track: all
   date: 3rd Oct
-  time: 19:00 pm
+  time: 19:00 IST
   type: special
   speaker:
 
 - title: PyCon India 2020 Closing Address
   track: all
   date: 3rd Oct
-  time: 20:00 pm
+  time: 20:00 IST
   type: special
+
+# Schedule for 4th Oct
+
+- title: Getting Started with Python
+  track: 1
+  date: 4th Oct
+  time: 11:30 IST
+  speaker: Evan Smith
+  hyperlink: https://in.pycon.org/cfp/2020/proposals/getting-started-with-python~b4x8V/
+
+- title: Python for Computational Social Science with Text, Networks and clever data games
+  track: 2
+  date: 4th Oct
+  time: 11:30 IST
+  speaker: Bhargav Srinivasa
+  hyperlink: https://in.pycon.org/cfp/2020/proposals/python-for-computational-social-science-with-text-networks-and-clever-data-games~dyPPE/
+
+- title: 
+  track: 3
+  date: 4th Oct
+  time: 11:30 IST
+  speaker: 
+  hyperlink: 
+
+- title: 
+  track: 4
+  date: 4th Oct
+  time: 11:30 IST
+  speaker: 
+  hyperlink: 
+
+- title: Break / Networking
+  track: all
+  date: 3rd Oct
+  time: 14:00 IST
+
+- title: Python on Kubernetes: A primer
```

```suggestion
- title: "Python on Kubernetes: A primer"
```
`:` is map operator in YAML. Strings containing `:` need to be wrapped with quotes.

That is why the build is failing.